packages/ocaml-secondary-compiler/ocaml-secondary-compiler.4.08.1-1/files/fix-gcc10.patch

```diff
@@ -0,0 +1,43 @@
+commit 3f10a16153308f967149917585d2bc0b9c06492c
+Author: Anil Madhavapeddy <anil@recoil.org>
+Date:   Sun Jun 21 18:40:27 2020 +0100
+
+    Add `-fcommon` unconditionally to CFLAGS to fix gcc10 build
+    
+    Signed-off-by: Anil Madhavapeddy <anil@recoil.org>
+
+diff --git a/configure b/configure
+index 9a78a4554..0c54b560b 100755
+--- a/configure
++++ b/configure
+@@ -2688,7 +2688,7 @@ mkexe="\$(CC) \$(OC_CFLAGS) \$(OC_CPPFLAGS) \$(OC_LDFLAGS)"
+ 
+ # Flags for building executable files with debugging symbols
+ mkexedebugflag="-g"
+-common_cflags=""
++common_cflags="-fcommon"
```

Comment:
This isn't in the `configure.ac` diff (and FWIW would break MSVC)?

Reply:
that snuck in! removed in the base compiler packages too